### DISCUSSION TITLE
Install problems due to wrong home directory

### DISCUSSION THREAD

**Discussion by moeretm** [NvChad/NvChad#2696] [Q&A] [answered]
When using the latest ubuntu snap of nvim (0.9.4) and version 2.0 of NvChad i run into an error saying:

```
Error detected while processing /snap/nvim/2819/usr/share/nvim/runtime/plugin/rplugin.vim[66]..function <SNR>12_LoadRemotePlugins[
1]..<SNR>12_GetManifest[1]..<SNR>12_GetManifestPath:
line   10:
E739: Cannot create directory /home/user@something/.local: permission denied
```

This error indicates that the home-directory path is constructed wrongly. For reasons beyond my control the username (that is reported by `whoami`) on my system is an email-address while the home-directory is just the name (i.e. "home/user/"), Is there any way to overwrite the path that is used when NvChad is trying to create the .local folder?

When closing neovim I get an additional error that is quite similar:
`E886: Failed to create directory /home/user@something/.local for writing ShaDa file: permission denied`

**Comment by siduck** [comment]
seems like nvim doesnt have perms to write on your filesystem, not a nvchad issue :/ . Please make it writable!

**Reply by moeretm** [reply]
That is not the problem. The problem is that at some point in the installation process the nvchad lua scripts construct the path to the home-directory by doing something like "/home/" + `whoami` + "/" which does not work on my system. Say my home directory is "/home/user/" but `whoami` returns "user@organization.net" then the nvchad scripts try to operate with "/home/user@organization.net/.local" which just can't work.

**Reply by siduck** [reply]
just clone the repo where your  ~/.config/nvim is supposed to be then?

**Reply by moeretm** [reply, marked as answer]
I actually identified the problem to be with `nvim` not `nvchad`. 
https://github.com/neovim/neovim/blob/0eaae1bc057a2a4672d3e485498137a0d2282ad3/runtime/plugin/rplugin.vim#L11
It seems to be possible to overwrite the path that is used. Among others I have to overwrite NVIM_RPLUGIN_MANIFEST
